app/common/patient/components/wizard/patientWizard.js

```diff
@@ -92,18 +92,22 @@
       patientService.createPatientProfile(vm.patient)
         .then(patientProfile => {
           notifier.success(translateFilter('COMMON_PATIENT_CREATED'));
-          $state.go('dashboard', {patientUuid: patientProfile.patient.uuid});
+          $state.go('dashboard', { patientUuid: patientProfile.patient.uuid });
         })
-        .catch(() => {
-          notifier.error(translateFilter('COMMON_MESSAGE_ERROR_ACTION'));
+        .catch(error => {
+          if (typeof error === "string") {
```

Comment:
From eslint:   

    98:15  error  You should use the angular.isString method  angular/typecheck-string

Why not use an object with a `message` property, like an exception?